src/components/OSCALLoader.js

```diff
@@ -4,7 +4,7 @@ import Split from "react-split";
 import { makeStyles } from "@material-ui/core/styles";
 import { Box, Fab } from "@material-ui/core";
 import CodeIcon from "@material-ui/icons/Code";
-import { populatePartialRestData } from "./oscal-utils/OSCALUtils";
+import { populatePartialRestData, buildRestRequestUrl, performRestRequest, restMethods, oscalObjectTypes } from "./oscal-utils/OSCALRestUtils";
```

Comment:
Maybe:

```ts
import * as restUtils from "./oscal-utils/OSCALRestUtils";
```

Or maybe a better local name than `restUtils`. It'd require some refactoring to use the namespace. But it also means that we could rename the actual functions to be `populatePartialData`, `buildRequestUrl`, `performRequest`, and `methods` since the whole "REST" thing is implied by the file/module name.